src/pell.js

```diff
@@ -22,6 +22,15 @@ const image = () => {
   if (url) execute('insertImage', ensureHTTP(url))
 }
 
+const video = () => {
+  const url = window.prompt('Enter the video URL')
+  const type = window.prompt('Enter the video type')
```

Comment:
If only one source is specified, the `<source>` element is not necessary. You can simply use `<video src="">` and omit the type